src/main/java/sa/assertj/iterables/AreAtLeastOne.java

```diff
@@ -0,0 +1,34 @@
+package sa.assertj.iterables;
+
+import org.assertj.core.api.Condition;
+import sa.assertj.Experiment;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AreAtLeastOne extends Experiment {
+    static Experiment.DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int i = 0; i < numOfSamples; i++) {
+            ArrayList<Integer> list = new ArrayList<>();
+            Random rand = new Random();
+            int element = rand.nextInt();
+            for (int j = 0; j < size; j++) {
+                list.add(element);
+            }
+
+            Condition<Integer> condition = new Condition<>(s -> s.equals(element), "equals");
```

Comment:
This can have the same form as the `AnyMatch` to make sure it doesn't always succeed after the first element.